src/resources/help/en_US/relnotes3.2.1.html

```diff
@@ -24,6 +24,7 @@
 <p><b><font size="4">Bugs fixed in version 3.2.1</font></b></p>
 <ul>
   <li>Fixed a bug that resulted in incorrect plots of mesh_quality/min_side_volume and mesh_quality/max_side_volume.</li>
+  <li>Fixed a few issues running build_visit with '--static'.</li>
```

Comment:
Do we normally bracket literals with single quotes in html or does `<code>--static</code>` work better?